src/static/css/splash.css

```diff
@@ -1,59 +1,230 @@
-/** NOTE: Bulk of the styles to be replaced soon in https://github.com/HTTPArchive/almanac.httparchive.org/issues/106 */
-
+/* Browser elements */
 body {
-    font-family: 'Open Sans', sans-serif;
-    color: #444;
+	font-family: 'Open Sans', sans-serif;
+	color: #444;
+	margin: 0;
+}
+
+h2 {
+	font-weight: inherit;
+}
+
+button {
+	background: #f7f879;;
```

Comment:
Implement hover and focus states. The yellow border button becomes a yellow background button and vice versa.

For example, from this style:
![image](https://user-images.githubusercontent.com/1120896/63056825-dcd85080-beb6-11e9-9121-7c7872433519.png)

to this style:
![image](https://user-images.githubusercontent.com/1120896/63056836-e3ff5e80-beb6-11e9-8984-dd0791625878.png)